docs/apis-tools/web-modeler-api/tutorial.md

```diff
@@ -0,0 +1,172 @@
+---
+id: modeler-api-tutorial
+title: Tutorial
+sidebar_position: 3
+slug: /apis-tools/web-modeler-api/tutorial
+description: "Step through an example to view your existing clients, create a client, view a particular client's details, and delete a client."
+---
+
+In this tutorial, we'll step through examples to highlight the capabilities of the Web Modeler API, such as creating and deleting a file.
+
+## Prerequisites
+
+- If you haven't done so already, [create a cluster](/components/react-components/create-cluster.md).
+- Upon cluster creation, create your first client by navigating to **Console > Organization > Administration API > Create new credentials**. Ensure you determine the scoped access for client credentials. For example, in this tutorial we will retrieve and delete a file. Ensure you check all the boxes for Web Modeler and Zeebe client scopes.
+
+:::note
+Make sure you keep the generated client credentials in a safe place. The **Client secret** will not be shown again. For your convenience, you can also download the client information to your computer.
+:::
+
+- In this tutorial, we utilize a JavaScript-written [GitHub repository](https://github.com/camunda/camunda-api-tutorials) to write and run requests. Clone this repo before getting started.
+- Ensure you have [Node.js](https://nodejs.org/en/download) installed as this will be used for methods that can be called by the CLI (outlined later in this guide). Run `npm install` to ensure you have updated dependencies.
+
+## Getting started
+
+- A detailed API description can be found [here](https://modeler.cloud.camunda.io/swagger-ui/index.html) via Swagger. With a valid access token, this offers an interactive API experience against your Camunda 8 cluster.
```

Comment:
> With a valid access token, this offers an interactive API experience against your Camunda 8 cluster.

I would reword this since this API isn't scoped for the cluster level.